libnymea/interfaces/alarm.json

```diff
@@ -0,0 +1,15 @@
+{
+    "description": "An interface for alarms, such as fire or intruder alarms.",
```

Comment:
What's the difference between alarm and notify action? Is this zigbee releated idenity action?

Reply:
Alarm would be an actual alarm, like a fire alarm, or intruder alert siren. Notify on the other hand sends a message with text. Then identify on a zigbee level only flashes the LED to identify a device.

Reply:
I'm not following what you're asking for... the notification interface has an action to send a text message with a title and a body. The alarm interface just has an action to sound the alarm, like a siren...

Yes, a fire sensor does sound the alarm on its own when it detects fire, however, having multiple fire sensors allows to sound the alarm on all of them even if just one detects fire. Or it can be used as a intruder siren too if a window/door sensor detects a breach while being armed.